docs: Clarify multi remote exec plane usage (#9)
keptn-contrib/helm-service: chart/README.md

| Parameter                               | Description                                                       | Default                                         |
|-----------------------------------------|-------------------------------------------------------------------|-------------------------------------------------|

> | `nameOverride`                          | Override the name label and distributor subscription name. <br />This option can be used to provide a unique name for multiple remote execution planes        | `""`                                            |

Reviewer: ```suggestion
| `nameOverride`                          | Override the name label and distributor subscription name. <br />This option should be used to provide a unique name per helm service in a multi remote execution plane setup.        | `""`                                            |
```
